.github/workflows/mi210.yml

```diff
@@ -4,90 +4,89 @@ on:
-#            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
-#            -DKokkos_ENABLE_SERIAL=ON \
-#            -DKokkos_ENABLE_HIP=ON \
-#            -DKokkos_ARCH_VEGA90A=ON \
-#            -DKokkos_ENABLE_TESTS=OFF \
-#            -DKokkos_ENABLE_EXAMPLES=OFF \
-#            -DCMAKE_VERBOSE_MAKEFILE=ON \
-#            -DCMAKE_CXX_EXTENSIONS=OFF \
-#            -DCMAKE_CXX_STANDARD=17 \
-#            -DBUILD_SHARED_LIBS=OFF \
-#            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
-#            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
-#            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
-#            ..
-#
-#      - name: build_and_install_kokkos
-#        working-directory: kokkos/build
-#        run: make -j16 install
-#
-#      - name: configure_kokkos_kernels
-#        run: |
-#          mkdir -p kokkos-kernels/{build,install}
-#          cd kokkos-kernels/build
-#          HIPCC=$(which hipcc)
-#          cmake -DCMAKE_CXX_COMPILER=$HIPCC \
-#            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
-#            -DCMAKE_CXX_FLAGS="-O3 " \
-#            -DCMAKE_INSTALL_PREFIX= \
-#            -DKokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
-#            -DKokkosKernels_ENABLE_TESTS=ON \
-#            -DKokkosKernels_ENABLE_PERFTESTS=ON \
-#            -DKokkosKernels_ENABLE_EXAMPLES:BOOL=ON \
-#            -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
-#            -DKokkosKernels_INST_COMPLEX_DOUBLE=ON \
-#            -DKokkosKernels_INST_DOUBLE=ON \
-#            -DKokkosKernels_INST_ORDINAL_INT=ON \
-#            -DKokkosKernels_INST_OFFSET_SIZE_T=ON \
-#            -DKokkosKernels_INST_OFFSET_INT=ON \
-#            -DKokkosKernels_INST_LAYOUTLEFT=ON \
-#            -DKokkosKernels_ENABLE_TPL_ROCSPARSE=OFF \
-#            -DKokkosKernels_ENABLE_TPL_ROCBLAS=OFF \
-#            -DKokkosKernels_ENABLE_TPL_CUSOLVER=OFF \
-#            -DKokkosKernels_ENABLE_TPL_CUSPARSE=OFF \
-#            -DKokkosKernels_ENABLE_TPL_CUBLAS=OFF \
-#            -DCMAKE_EXE_LINKER_FLAGS="" \
-#            -DBUILD_SHARED_LIBS=OFF \
-#            -DKokkosKernels_ENABLE_DOCS=OFF \
-#            ..
-#
-#      - name: build
-#        working-directory: kokkos-kernels/build
-#        run: make -j12 all
-#
-#      - name: test
-#        working-directory: kokkos-kernels/build
-#        run: ctest --output-on-failure -V --timeout 3600
+  PR_VEGA90A_ROCM561_HIP_SERIAL_LEFT:
+    name: PR_VEGA90A_ROCM561_HIP_SERIAL_LEFT
+    runs-on: [kk-env-openblas-0.3.23-hip-5.6.1-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v4
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v4
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          echo "GITHUB_WORKSPACE = $GITHUB_WORKSPACE"
+          mkdir "$GITHUB_WORKSPACE/kokkos/build"
```

Comment:
Thanks for the work on this @lucbv !
When running this job through GA, I just wanted to check that the `GITHUB_WORKSPACE` directory is unique to each job (the name includes for example a sha, timestamp, or contains a PR number)?
If not (and if there is not clean up of previous jobs) then the `mkdir` lines could trigger an error if the directory already exists (adding a `-p` would avoid the error). I'm assuming this should not occur, but wanted to check for my own understanding

Reply:
Good question, each jobs runs in its own container image, here it happens to be the same but it gets reloaded each time. Also since we have two nodes to test on caraway these might actually run on two different compute nodes concurrently. I saw them running at the same time so in this particular PR that's definitely what happened.
The downside of this approach is that you cannot go back and see the artifacts from the container, you have to rerun the pipeline in an interactive container yourself to observe the error... we could save the image after the container is done running the tests and save it somewhere but I think we would use a lot of space so we are not doing that for now...

Reply:
Ah right, thanks! Yeah, this is within the container, so no worries 👍